worker/docker.go

```diff
@@ -110,7 +111,8 @@ func (m *DockerManager) Borrow(ctx context.Context, pipeline, modelID string) (*
 	if !ok {
 		// The container does not exist so try to create it
 		var err error
-		rc, err = m.createContainer(ctx, pipeline, modelID, false)
```

Comment:
@yondonfu I did not yet implement the ability to use optimization flags for dynamically loaded containers since `SFAST` results in significant startup delays. We can enable them later for other optimizations like https://github.com/NVIDIA/TensorRT.